tests/Unit/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/Test_BjorhusImpl.cpp

```diff
@@ -209,10 +193,48 @@ void test_constraint_preserving_bjorhus_v_zero(
 }
 }  // namespace
 
+namespace {
+template <size_t VolumeDim>
+tnsr::iaa<DataVector, VolumeDim, Frame::Inertial> wrapper_func_v_zero(
+    const tnsr::I<DataVector, VolumeDim, Frame::Inertial>&
+        interface_normal_vector,
+    const tnsr::iaa<DataVector, VolumeDim, Frame::Inertial>&
+        four_index_constraint,
+    const tnsr::a<DataVector, 3, Frame::Inertial>& char_speeds) {
+  std::array<DataVector, 4> char_speed_array{
+      get<0>(char_speeds), get<1>(char_speeds), get<2>(char_speeds),
+      get<3>(char_speeds)};
+  auto dt_v_zero =
+      make_with_value<tnsr::iaa<DataVector, VolumeDim, Frame::Inertial>>(
+          get<0>(interface_normal_vector), 0.);
+  GeneralizedHarmonic::BoundaryConditions::Bjorhus::
+      add_constraint_preserving_terms_to_dt_v_zero<VolumeDim, DataVector>(
+          make_not_null(&dt_v_zero), interface_normal_vector,
+          four_index_constraint, char_speed_array);
+  return dt_v_zero;
+}
+
+template <size_t VolumeDim>
+void test_add_constraint_preserving_terms_to_dt_v_zero(
+    const size_t grid_size_each_dimension) noexcept {
+  pypp::check_with_random_values<1>(
+      &wrapper_func_v_zero<VolumeDim>,
+      "Evolution.Systems.GeneralizedHarmonic.BoundaryConditions.Bjorhus",
+      "add_constraint_preserving_terms_to_dt_v_zero", {{{-1., 1.}}},
+      DataVector(grid_size_each_dimension));
+}
+}  // namespace
+
 SPECTRE_TEST_CASE("Unit.Evolution.Systems.GeneralizedHarmonic.BCBjorhus.VZero",
                   "[Unit][Evolution]") {
+  pypp::SetupLocalPythonEnvironment local_python_env{""};
```

Comment:
Same suggestion about env from the other PR -- it could be the directory here and then the module can just be `"Bjorhus"` in the `check_with_random_values` call.

Reply:
This has been done for the same reason as stated [here](https://github.com/sxs-collaboration/spectre/pull/2998#discussion_r636059987)